### PR TITLE
Revert PV and statefulset related changes

### PR DESCRIPTION
**Pull request by maimaisie**
###### Description

Reverting since we have another minor release to make

###### Testing performed

- [ ] ci/build.sh
- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

File: deploy/kubernetes/fluentd-sumologic.yaml.tmpl [hunk 1]
```diff
@@ -13,14 +13,14 @@ data:
     @include common.conf
     @include metrics.conf
     @include logs.conf
-  buffer.output.conf: |-
+  buffer.output.conf: |
```
Comment (maimaisie) [inline deploy/kubernetes/fluentd-sumologic.yaml.tmpl]: hmm not sure where this change is coming from

Comment (rvmiller89) [reply deploy/kubernetes/fluentd-sumologic.yaml.tmpl]: maybe the addition of the newline character in `buffer.output.conf`? Looks like it didn't have a trailing newline before

Comment (maimaisie) [reply deploy/kubernetes/fluentd-sumologic.yaml.tmpl]: oh interesting, let me try removing it

Comment (maimaisie) [reply deploy/kubernetes/fluentd-sumologic.yaml.tmpl]: that fixed it 👀 thanks Ryan

**Pull request merged**